iexfinance/tests/test_altdata.py

```diff
@@ -23,7 +23,7 @@ def test_crypto_quote_list(self):
             get_crypto_quote(["BTCUSDT", "BAD"])
 
     def test_crypto_quote_json(self):
-        data = get_crypto_quote("BTCUSDT")
+        data = get_crypto_quote("BTCUSDT", output_format='json')
```

Comment:
Revert this per earlier comment

Reply:
Took care of that. All should be good now.